RestaurantOrderingSystem/src/main/java/spring/controllers/OrderController.java

```diff
@@ -2,31 +2,51 @@
 
 import java.util.ArrayList;
 
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
 import spring.models.MenuItem;
 import spring.models.Order;
 import spring.models.OrderHistory;
 import spring.models.OrderItem;
 import spring.models.OrderMemento;
+import spring.models.RealRestaurant;
 import spring.models.Restaurant;
 import spring.models.User;
+import utils.DBManager;
 
+@Controller
 public class OrderController {
 	
 	private Order order;	
 	private User customer;
-	private Restaurant restaurant;
+	private Restaurant restaurant = new RealRestaurant();

```

Comment:
this should be a proxy restaurant so that we can control access.  Ideally the RealRestaurant will not exist outside of a proxy restaurant.